indexer-opts/README.md

```diff
@@ -0,0 +1,52 @@
+# indexer-opts - Opts struct for CLI arguments for microindexers
+
+This lib is a shared `clap::Parser` instance that defines CLI arguments for all microindexers.
+
+### Features:
+
+- Basic arguments for indexer to start
+- `init_tracing` for initializing the logs for indexer
+- `StartMode` that allows to choose the strategy for restarts:
+  - `from-latest`
+  - `from-interruption` (default)
+
+### Parameters
+
+- `indexer-id` | **Required** Sets the micro-indexer instance ID (for reading/writing indexer meta-data)
+- `start-block-height` | **Required** Block height to start the stream from (not used if `start_mode == from-latest`)
+- `near-archival-rpc-url` | **Required** NEAR JSON RPC URL
+- `chain-id` | **Required** Chain id: testnet or mainnet, used for NEAR Lake initialization
+- `port` | Default: 3000 Port to enable metrics/health service
+- `start-mode` | Default: "from-interruption" Start mode for instance (`from-interruption`, `from-latest`)
+- `database-url` | **Required** Database URL
+
```

Comment:
I guess we want to add `end_block_height` for historical indexers. The block when we want to stop the process.

Reply:
We may add everything we want here and ready to support by indexers. I believe it is out of the scope of this PR and task :)

Reply:
We don't have an issue related to this PR, so it's hard to say what was the initial purpose of the PR.

I thought it was the PR to solve 2 issues:
- clean up the mess with the parameters, move the general logic to the separate library
- automate the data collection for historical indexers

Without `end_block_height`, we don't solve the second point.
Without second point, why do we add this new table here and do other complex things